schedule/tests/test_models.py

```diff
@@ -183,18 +202,41 @@ def testing_behavior(self, start_date, diff, employees, shift_pattern,
         start_date = datetime.date(2019, 9, 4)  # wednesday
         diff = datetime.timedelta(days=3)
         expected_status = [False, False, False]
-        expected_not_registered = [[(datetime.date(2019, 9, 4), night_shift)],
-                                   [(datetime.date(2019, 9, 7), night_shift),
-                                    (datetime.date(2019, 9, 8), night_shift)],
-                                   [(datetime.date(2019, 9, 10), night_shift)]]
+        expected_status_detail = [{'overridable': [],
```

Comment:
Not sure if you could just create a method to construct some of these larger variables in your test cases, this way you don't need to keep copy pasting the declaration and it'll be easier to add future test cases.

Reply:
but the method that needs to create those is the method im testing